zboxcore/sdk/chunked_upload_chunk_reader.go

```diff
@@ -76,8 +77,14 @@ func createChunkReader(fileReader io.Reader, size, chunkSize int64, dataShards i
 		return nil, errors.Throw(constants.ErrInvalidParameter, "hasher")
 	}
 
+	fragmentSize := int64(dataShards) * chunkSize
+	numFragments := int(math.Ceil(float64(size) / float64(fragmentSize)))
+	resultantSize := int64(numFragments) * fragmentSize
+	zPaddings := resultantSize - size
+	newReader := io.MultiReader(fileReader, bytes.NewBuffer(make([]byte, zPaddings)))
```

Comment:
do you test it on `./zbox feed` , `./zbox stream` and `./zbox download --live`? if not, please test it. Because I am not sure if this fixing works with all of them.

Reply:
let me know if manual tests are passed. I will get this PR approved.